src/main/webapp/resources/js/pages/user/components/UserDetailsPage.jsx

```diff
@@ -0,0 +1,196 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import {
+  Button,
+  Form,
+  Input,
+  notification,
+  Select,
+  Space,
+  Switch,
+  Typography,
+} from "antd";
+import { formatDate } from "../../../utilities/date-utilities";
+
+import {
+  useGetUserDetailsQuery,
+  useEditUserDetailsMutation,
+} from "../../../apis/users/users";
+
+/**
+ * React component to display the user details page.
+ * @returns {*}
+ * @constructor
+ */
+export default function UserDetailsPage() {
+  const { userId } = useParams();
+  const { data, isSuccess } = useGetUserDetailsQuery(userId);
+  const [editUser] = useEditUserDetailsMutation();
+
+  const [formErrors, setFormErrors] = useState();
+
+  const onFormFinish = (values) => {
+    editUser({ userId: userId, ...values })
+      .unwrap()
+      .then((payload) => {
```

Comment:
Looks like the payload here isn't being used..you can just the empty parentheses here.

Reply:
Agreed, but when I try removing it, the hook doesn't work properly.